.evergreen/generate.yml

```diff
@@ -19,6 +26,19 @@ functions:
       content_type: text/plain
       permissions: public-read
 
+  attach-e2e-generated:
```

Comment:
[nit] I know I implemented this originally, but maybe having `attach-generated` and `attach-e2e-generated` would make more sense in `attach.yml`?